resources/aws/route_table.go

```diff
@@ -112,3 +112,47 @@ func (r *RouteTable) Delete() error {
 func (r RouteTable) ID() string {
 	return r.id
 }
+
+// MakePublic creates a route that allows traffic from outside the VPC.
+// To do that, it needs to add a route on the Internet Gateway of the VPC.
+func (r RouteTable) MakePublic() error {
+	gatewayID, err := r.getInternetGateway()
+	if err != nil {
+		return microerror.MaskAny(err)
+	}
+
+	if _, err := r.Client.CreateRoute(&ec2.CreateRouteInput{
+		RouteTableId:         aws.String(r.id),
+		DestinationCidrBlock: aws.String("0.0.0.0/0"),
+		GatewayId:            aws.String(gatewayID),
+	}); err != nil {
+		return microerror.MaskAny(err)
+	}
+
+	return nil
+}
+
+// An internet gateway is what enables communication between a VPC and the outside Intenet.
+// https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Internet_Gateway.html
+func (r RouteTable) getInternetGateway() (string, error) {
+	resp, err := r.Client.DescribeInternetGateways(&ec2.DescribeInternetGatewaysInput{
+		Filters: []*ec2.Filter{
+			&ec2.Filter{
+				// retrieve only the gateway attached to the vpc of the RouteTable r
```

Comment:
The `r` at the end confused me because there is no fullstop at the end of the sentence. I realized now that `r` is the variable for the instance of the rout table. My shortcoming. :D Please make a sentence out of that and go ahead.